src/cmd_line/commands/file.ts

```diff
@@ -3,9 +3,10 @@ import * as node from '../node';
 import * as path from 'path';
 import * as util from 'util';
 import * as vscode from 'vscode';
-import untildify = require('untildify');
 import { Logger } from '../../util/logger';
 
+const untildify: typeof import('untildify') = require('untildify');
```

Comment:
unlike esm import https://github.com/VSCodeVim/Vim/pull/4059/files#r322551630 , this still uses cjs require as replacement of `import = require` syntax: this 3rd party dep uses export default, either requires enable tsconfig `esmoduleInterop` or use cjs. instead of changing global config, import specific modules only via cjs.

Reply:
I'm not super well-versed in this stuff. My understanding is that `import` is needed rather than `require` because `import` can't be done dynamically anywhere, so static analysis like tree shaking is more possible? If that's the case, do we get the benefit of tree shaking, etc. for modules imported like this?

Reply:
This `import` syntax is typescript specific: assign `typeof` imported module to assigned var (`const untildify`) from cjs `require`. Ts doesn't import types from cjs require, but allow import types via this syntax as needed - and runtime code will not contain this as same as other type information does.

Reply:
This not cause of tree shaking: as described in comment, it's way to import `default` export modules without enabling tsconfig's esmoduleinterop.

Reply:
For these modules I didn't try hard to enable esm import: it's being used lot / small and no huge gain via tree shaking as far as I can tell.